frontend/src/components/auth/AuthActions.tsx

```diff
@@ -17,6 +17,9 @@ const AuthActions = (): React.ReactElement => {
 
   const mode = searchParams.get("mode");
   const uid = searchParams.get("uid");
+  const isNewAccount = searchParams.get("new-account")
+    ? searchParams.get("new-account")?.toLowerCase() === "true"
+    : null;
```

Comment:
is there any way we can read the parameter as a bool instead of comparing it to a string

Reply:
Don't think so. I think we'll have to use a third party library which I don't think is worth it:
https://stackoverflow.com/questions/39599012/proper-way-to-parse-boolean-query-string-param-in-node-express